From 7bd8575b65e6a7ab9954cf30dcaff398bf9246e9 Mon Sep 17 00:00:00 2001
From: Shannon McPherson <shannon@lunarg.com>
Date: Mon, 10 Sep 2018 11:13:02 -0600
Subject: [PATCH] scripts: Modify scripts to accomodate new exts

Parameter validation was not gathering and processing structure
information correctly for one of the 84 Headers extensions
---
 scripts/parameter_validation_generator.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scripts/parameter_validation_generator.py b/scripts/parameter_validation_generator.py
index 8bb8da382..b157a1578 100644
--- a/scripts/parameter_validation_generator.py
+++ b/scripts/parameter_validation_generator.py
@@ -711,6 +711,7 @@ def genVkStructureType(self, typename):
         # Add underscore between lowercase then uppercase
         value = re.sub('([a-z0-9])([A-Z])', r'\1_\2', typename)
         value = value.replace('D3_D12', 'D3D12')
+        value = value.replace('ASTCDecode', 'ASTC_Decode')
         value = value.replace('Device_IDProp', 'Device_ID_Prop')
         value = value.replace('LODGather', 'LOD_Gather')
         value = value.replace('Features2', 'FEATURES_2')
